src/main/scala/QuickTest.scala

```diff
@@ -0,0 +1,18 @@
+/**
+  * Created by Gerard on 10/05/2016.
+  */
+
+
+import scala.collection.mutable
+
+
+

```

Comment:
Could you remove the double blank lines, leave maximum one between the statement.